src/services/rfqm_service.ts

```diff
@@ -296,4 +339,87 @@ export class RfqmService {
             orderHash,
         };
     }
+
+    public async submitMetaTransactionSignedQuoteAsync(
+        params: MetaTransactionSubmitRfqmSignedQuoteParams,
+    ): Promise<MetaTransactionSubmitRfqmSignedQuoteResponse> {
+        const metaTransactionHash = params.metaTransaction.getHash();
+
+        // check that the firm quote is recognized as a previously returned quote
+        const quote = await this.dbUtils.findQuoteByMetaTransactionHashAsync(metaTransactionHash);
+        if (quote === undefined) {
+            RFQM_SIGNED_QUOTE_NOT_FOUND.inc();
+            throw new NotFoundError('quote not found');
+        }
+
+        // validate that the expiration window is long enough to fill quote
+        const currentTimeMs = new Date().getTime();
+        if (
+            !params.metaTransaction.expirationTimeSeconds
+                .times(ONE_SECOND_MS)
+                .isGreaterThan(currentTimeMs - RFQM_MINIMUM_EXPIRY_DURATION_MS)
```

Comment:
Isn't this `+ RFQM_MINIMUM_EXPIRY_DURATION_MS `?

Reply:
yep, typo